test/integration/test_upload_configuration_options.py

```diff
@@ -64,6 +67,24 @@ def test(self):
         # the newer API decorator that handles those details.
         assert create_response.status_code >= 400
 
+    @skip_without_datatype("velvet")
+    def test_composite_datatype(self):
```

Comment:
Should this method be called `test_secondary_file` (or alternatively `test_blocked_url_secondary_file` renamed to `test_blocked_url_composite_datatype`)?